lib/grpc/channel.ex

```diff
@@ -10,18 +10,20 @@ defmodule GRPC.Channel do
     * `:host` - server's host to connect
     * `:port` - server's port to connect
     * `:scheme` - scheme of connection, like `http`
+    * `:cred` - credentials used for authentication
     * `:adapter` - a client adapter module, like `GRPC.Adapter.Chatterbox.Client`
-    * `:payload` - the payload needed by the adapter
+    * `:client` - an instance of a client adapter, like `%GRPC.Adapter.Chatterbox.Client{}`
```

Comment:
It looks like `Stream` also has a payload. I kept getting confused by the name `payload` here. It seems like what this really is is the state of the client. In the case of chatterbox we can get away with just storing the `pname` in the client state.